backend/src/api/routes/config/apply.ts

```
@@ -0,0 +1,45 @@
+      applyConfig({
+        config: body.config,
+        dry_run: body.dry_run ?? false,
+        prune: body.prune ?? false,
+      })
+    );
+    successResponse(res, result);
+  } catch (err) {
+    if (err instanceof ConfigValidationError) {
+      res
+        .status(400)
+        .json({ error: 'config_validation', message: err.message, path: err.path });
```

ESLint

[error] Replace `⏎········.status(400)⏎········` with `.status(400)`

(prettier/prettier)